vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -448,6 +449,9 @@ bool FrameworkElementViewManager::UpdateProperty(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Expanded)]);
       DynamicAutomationProperties::SetAccessibilityStateCollapsed(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)]);
+      if (auto peer = xaml::Automation::Peers::FrameworkElementAutomationPeer::FromElement(element)) {
+        peer.RaiseAutomationEvent(xaml::Automation::Peers::AutomationEvents::LiveRegionChanged);
+      }
```

Comment:
I think that there may be cases where we are given an `accessibilityState` that was the same as the previous one. Could you add a check that the new value is different from previous before firing the change event?

Reply:
Also, we shouldn't be firing these events on the first set -- since they didn't change if we are just adding the element to the tree.

Reply:
I was originally thinking the same thing also, but I think that LiveRegionChanged must have some logic to handle those cases because narrator only announces when the accessibilityState is actually changed(although I can't find documentation/info to confirm this).

Note we're doing something very similar to this for accessibilityHint with: 
https://github.com/microsoft/react-native-windows/blob/537f6d7ac9154ea1bd4bf767075d7abe3a4f17e0/vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp#L261
and 

https://github.com/microsoft/react-native-windows/blob/537f6d7ac9154ea1bd4bf767075d7abe3a4f17e0/vnext/Microsoft.ReactNative/Utils/AccessibilityUtils.cpp#L15-24